README-UK.md

```diff
@@ -18,15 +18,18 @@
 
 [🇬🇧 English](./README.md) | 🇺🇦 Українська
 
-- [Альтернатива `mqpacker`](#альтернатива-mqpacker)
-- [Використання в CSS-in-JS 🚀](#використання-в-css-in-js-)
-- [Встановлення](#встановлення)
-- [Використання](#використання)
-   - [mobile-first](#mobile-first)
-   - [desktop-first](#desktop-first)
-- [Конфігурація сортування](#конфігурація-сортування)
-   - [Параметри конфігурації](#параметри-конфігурації)
-- [Інформація про проєкт](#інформація-про-проєкт)
+- [sort-css-media-queries](#sort-css-media-queries)
+  - [Зміст](#зміст)
+  - [Альтернатива `mqpacker`](#альтернатива-mqpacker)
+  - [Використання в CSS-in-JS 🚀](#використання-в-css-in-js-)
+  - [Встановлення](#встановлення)
+  - [Використання](#використання)
+    - [mobile-first](#mobile-first)
+    - [desktop-first](#desktop-first)
+  - [Конфігурація сортування](#конфігурація-сортування)
+    - [Параметри конфігурації](#параметри-конфігурації)
+      - [`unitlessMqAlwaysFirst`](#unitlessmqalwaysfirst)
+  - [Інформація про проєкт](#інформація-про-проєкт)
```

Comment:
Here my auto Markdown formatting seems to have extended the table of contents.

Personally, I find this somewhat unnecessary in this context.